apps/prairielearn/src/lib/editors.js

```diff
@@ -167,10 +167,18 @@ class Editor {
               }
 
               try {
+                // Remember that we attempted a push. If the push fails, we'll retain
+                // the `false` value.
+                job.data.pushed = false;
```

Comment:
See https://github.com/PrairieLearn/PrairieLearn/pull/7966#discussion_r1244479860 for a discussion of if we should use separate flags for "did attempt" and "did succeed".

Reply:
As I commented there, I (weakly) prefer separate flags.

Reply:
I'll take it! Done in #343b850.